src/js/components/modal.js

```diff
@@ -0,0 +1,29 @@
+export class cookies {
+    $bodyElement = document.querySelector('body');
+    bodyOverlayClassName = 'overlay';
+    hideClassName = 'hide';
+    showModalClassName = 'show';
+    
+
+    constructor(modalName) {
+        this.modal = document.querySelector(`#${modalName}`);
+        this.closeButton = this.modal.querySelector('button');
+        this.localStorage = localStorage.getItem("cookiesAccepted");
```

Comment:
nie przypisuj localStorage do this, nie chcemy tego przypisywac do tej klasy, z localstorage zawsze korzysta sie tak

localStorage.set / get

wogole nie potrzebujesz tego w konstruktorze